### PR TITLE
chore: sort imports

### PR DESCRIPTION
**Pull request by BobTheBuidler**

### CODE REVIEW

File: backend.py [hunk 1]
```diff
@@ -1,13 +1,146 @@
...
+CORS(app)
 
-@app.route("/")
+@app.route("/", methods=['POST'])
 def hello_yrs_backend():
-    with db_session:
-      result = db.select('NOW()')
-      return f"<p>Hello, yrs-backend!</p><p>it's {result[0]}</p>"
+    def process_sale():
+        nonlocal taxable_events
```
Comment (mariuspod) [inline backend.py]: what's the purpose of `nonlocal` here ?

File: backend.py [hunk 2]
```diff
@@ -1,13 +1,146 @@
...
+        nonlocal unspent_lots
```
Comment (mariuspod) [inline backend.py]: same as above

File: backend.py [hunk 3]
```diff
@@ -1,13 +1,146 @@
...
 app = Flask(__name__)
+CORS(app)
 
-@app.route("/")
+@app.route("/", methods=['POST'])
 def hello_yrs_backend():
-    with db_session:
-      result = db.select('NOW()')
-      return f"<p>Hello, yrs-backend!</p><p>it's {result[0]}</p>"
+    def process_sale():
```
Comment (mariuspod) [inline backend.py]: do you need this inner func ?

File: inputs.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+def method_input():
+    choices = ['FIFO','LIFO']
+    assert request.json['type'] in choices
+    return request.json['type']
+
+
+def address_inputs():
+    good_list, bad_list = [], []
+    for address in request.json['addresses']:
+        clean = clean_address(address)
```
Comment (mariuspod) [inline inputs.py:14]: no need for extra variable `clean` here, you can just inline

File: inputs.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    return (good_list, bad_list)
+
+
+def clean_address(address: str):
+    if is_address(address): return convert.to_address(address)
+    
+    
+def is_address(address: str) -> bool:
+    return all([
+        len(address) == 42
```
Comment (mariuspod) [inline inputs.py:26]: I'd go for the validations from brownie.convert which are more robust.
Consider this:
`convert.to_address('0x' + '.' * 40) // len(address) == 42`

you could remove #is_address altogether and move up to the #clean_address with:
```
try:
    return convert.to_address(address)
except ValueError
    return None
```

then you can can check for:
```
if clean_address(address): ....
else: ...
```

**Review by mariuspod — COMMENTED** [review]
added some suggestions

File: backend.py (hunk 1 above)
Comment (BobTheBuidler) [reply backend.py]: I've very significantly restructured the yrs backend, many of these will have been changed by now. 
nonlocal taxable_events allows process_sale() to modify the taxable_events variable even though it resides outside of the function

File: backend.py (hunk 3 above)
Comment (BobTheBuidler) [reply backend.py]: at this point I think yes

File: inputs.py (hunk 5 above)
Comment (BobTheBuidler) [reply inputs.py:26]: we also use the brownie convert function as a check elsewhere in the code, just not in this part
if brownie is unable to convert to checksum, the backend will fail

Comment (mariuspod) [reply inputs.py:26]: the only thing, I'd always prefer the target system's validation over self-built ones.
So usually if you use brownie's convert address to determine if the address is valid, the validations will be more robust than re-inventing the wheel...

**Pull request merged**